app/graphql/types/mutation_type.rb

```diff
@@ -1,4 +1,5 @@
 module Types
   class MutationType < Types::BaseObject
+    field :create_user, mutation: Mutations::Users::CreateUser
```

Comment:
I've noticed this line in both your PR and Matt's most recent one; is this coming from another branch? No change necessary but curious.

Reply:
This is necessary for our Create User mutation that we made together. I accidentally deleted it when trying to clean up the branch of my filter method code. I posted in the channel about it so that Matt and Carleigh would know to add it back in.

Reply:
Got it, thanks!